lib/ansible/modules/cloud/azure/azure_rm_mysqldatabase.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_mysqldatabase
+version_added: "2.5"
+short_description: Manage MySQL Database instance.
+description:
+    - Create, update and delete instance of MySQL Database.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    server_name:
+        description:
+            - The name of the server.
+        required: True
+    name:
+        description:
+            - The name of the database.
+        required: True
+    charset:
+        description:
+            - The charset of the database. Check MySQL documentation for possible values.
+    collation:
+        description:
+            - The collation of the database. Check MySQL documentation for possible values.
+    force_update:
+      description:
+          - Needs to be set to True in order to MySQL Database to be updated.
```

Comment:
Are you able to expand on this a bit further, you should state that when it is true and the database does not match the config, then it is deleted and recreated.